src/components/space/space.tsx

```diff
@@ -27,12 +27,17 @@ export const Space = () => {
       </Suspense>
       {satellites.map((satellite) => (
         <Fragment key={satellite.id}>
-          <Satellite color={satellite.color} tle={satellite.tle} />
+          <Satellite 
+            color={satellite.color}
+            path={satellite.path}
+            tle={satellite.tle}
+            visible={satellite.visible} />
```

Comment:
Since the `Satellite` component doesn't need the `path` property, it should be removed wherever the component is used.

```suggestion
          <Satellite 
            color={satellite.color}
            tle={satellite.tle}
            visible={satellite.visible} />
```